package.json

```diff
@@ -1,35 +1,27 @@
 {
   "name": "@biscuit-auth/biscuit-wasm",
   "description": "WebAssembly wrapper for Biscuit authorization tokens",
-  "version": "0.3.0",
+  "version": "0.4.0",
   "license": "Apache-2.0",
   "repository": {
     "type": "git",
     "url": "https://github.com/biscuit-auth/biscuit-wasm"
   },
   "scripts": {
-    "build-esm": "wasm-pack build --target bundler --out-dir module --out-name biscuit --scope biscuit-auth",
-    "build-cjs": "wasm-pack build --target nodejs --out-dir dist --out-name biscuit --scope biscuit-auth",
-    "remove-pkg-cruft": "rm module/package.json dist/package.json dist/.gitignore module/.gitignore dist/README.md module/README.md",
-    "prepare-package": "npm run build-esm && npm run build-cjs && npm run remove-pkg-cruft && cp performance.js dist/snippets/*/inline0.js"
+    "build": "wasm-pack build --target nodejs --out-dir dist --out-name biscuit --scope biscuit-auth",
+    "remove-pkg-cruft": "rm dist/package.json dist/.gitignore dist/README.md dist/LICENSE",
+    "prepare-package": "npm run build && npm run remove-pkg-cruft && cp performance.js dist/snippets/*/inline0.js"
   },
   "files": [
-    "module/biscuit_bg.wasm",
-    "module/biscuit.js",
-    "module/biscuit.d.ts",
-    "module/snippets",
-    "dist/biscuit_bg.wasm",
     "dist/biscuit.js",
-    "dist/biscuit_bg.js",
     "dist/biscuit.d.ts",
+    "dist/biscuit_bg.js",
+    "dist/biscuit_bg.wasm",
+    "dist/biscuit_bg.wasm.d.ts",
     "dist/snippets"
   ],
+  "type": "commonjs",
   "main": "dist/biscuit.js",
-  "exports": {
```

Comment:
right, now that I remember, the point was to make a package that can be imported both in frontend js and in node, and the solution was suggested by @abernix. Can we keep that behaviour?